### DISCUSSION TITLE
Project still active??

### DISCUSSION THREAD

**Discussion by s-no1ukno** [MelkeyOSS/astrostation#315] [General]
Just wanted to reach out and see if this project is still active, and if not, if there's any help needed or ways to make it active again.  I'd love to contribute to an open source project like this, and I really love the vibe and feel of Astrostation - I think there's been some other parties interested in contributing as well.  

Hopefully we can get this rolling again!

**Comment by Melkeydev** [comment]
Hey!
Yes the project is active - well, kind of. Its in maintenance mode - so no active dev but thats only because I got swamped with other projects!!
If you or these other parties wanna contribute - id love to chat! Add you as contributors. 
The best way to reach me is via Discord, in the server I own we have a public channel for astrostation.

**Reply by s-no1ukno** [reply]
Excellent!  I'll jump in the discord as soon as I track down your link 💯  Thanks for the quick reply!

**Comment by disbh** [comment]
Hello! Is the Discord channel publicly accessible?

**Reply by Melkeydev** [reply]
https://discord.gg/melkeydevhouse

There ya go
